docker-compose.yml

```diff
@@ -5,5 +5,55 @@ services:
     build:
       context: .
       dockerfile: service.Dockerfile
+    volumes:
+      - ${HOST_PATH}:/./data
     ports:
       - "50061:50061"
+  batch-prover:
+    build:
+      context: .
+      dockerfile: batch-prover.Dockerfile
+    volumes:
+      - ${HOST_PATH}:/./data
+    environment:
+      - SCHEDULER_ADDR=http://127.0.0.1:50061
+  scheduler-test:
+    build:
+      context: .
+      dockerfile: scheduler-test.Dockerfile
+    volumes:
+      - ${HOST_DATA_PATH}:/app/data
+      - ${HOST_WORK_BASE}:/./starkjs
```

Comment:
You need download the starkjs and [stark-circuits](https://github.com/0xEigenLabs/eigen-zkvm/tree/main/stark-circuits) from the scratch, not mapping the one from your computer!

Reply:
ok